zenodo/modules/records/serializers/fields/sanitizedunicode.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2016 CERN.
+#
+# Zenodo is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Sanitized Unicode string field."""
+
+from __future__ import absolute_import, print_function
+
+from ftfy import fix_text
+
+from .trimmedstring import TrimmedString
+
+
+class SanitizedUnicode(TrimmedString):
+    """String field that sanitizes and fixes problematic unicode characters."""
+
+    UNWANTED_CHARACTERS = {
```

Comment:
1. Are those the only problematic characters that can occur or just some examples of characters we had in the data that caused problems?
2. Does the ftfy solve the serialization issue without this explicit removal of characters?